packages/react-components/src/organisms/__tests__/SearchControls.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import SearchControls from '../SearchControls';
+import { noop } from '../../utils';
+
+const props: ComponentProps<typeof SearchControls> = {
+  query: '',
+  placeholder: '',
+  onChangeSearch: noop,
+  filterOptions: [],
+  filterTitle: '',
+};
+it('renders the search controls', () => {
+  const { getByRole } = render(<SearchControls {...props} />);
+  expect(getByRole('textbox')).toBeVisible();
+  expect(getByRole('button')).toBeVisible();
+});
+
+it('Passes query correctly', () => {
+  const { getByRole } = render(<SearchControls {...props} query="test123" />);
+  expect(getByRole('textbox')).toHaveValue('test123');
+});
+
+it('shows and hides the dropdown menu', () => {
+  const { getByRole, getByText } = render(
+    <SearchControls {...props} query="test123" filterTitle="Filter Dropdown" />,
+  );
+  const filterButton = getByRole('button');
+  userEvent.click(filterButton);
+  expect(getByText('Filter Dropdown')).toBeVisible();
+});
```

Comment:
Menu auto-close on filter change would be good to test.
Slightly odd behavior though - maybe first check if this is actually a requirement, it seems like it could be considered annoying rather than helpful

Reply:
This does not hide automatically. I am just testing the show hide dropdown button.

Reply:
Then what is `filterOptions` doing in the `useEffect` hook dependencies?